Header  header
float32[] s_begin_mu_segments
float32[] mu_segment_values
